src/components/UseCaseQuotes/use-case-customers.module.css

```
@@ -1,8 +1,9 @@
 .use-case-customers {
+  max-width: 100%;
   padding: 2.6rem 0 0 0;
   border-radius: 16px;
   background: var(--theme-card-bg-color);
-  flex: 1;
+  flex: 0.9;
```

```suggestion
  flex: 1;
```

unsure why `0.9` is used, i would keep `1`. It looks just as good